configs/components/gettext.rb

```diff
@@ -0,0 +1,48 @@
+component "gettext" do |pkg, settings, platform|
+  pkg.version "0.19.8.1"
+
+  if platform.is_windows?
+    # Compilation requires building iconv (that works) and a bunch of patches to gettext.
+    # For now, punt and use a pre-built package.
+    if platform.architecture == "x64"
+      arch = "64"
+      pkg.md5sum "f877ebff42736535415b4ddb9c631b86"
+    else
+      arch = "32"
+      pkg.md5sum "bced59a375aef8d325b746f0dfbe8f48"
+    end
+
+    pkg.url "https://github.com/mlocati/gettext-iconv-windows/releases/download/v#{pkg.get_version}-v1.14/#{pkg.get_name}#{pkg.get_version}-iconv1.14-shared-#{arch}.zip"
+
+    pkg.environment("PATH", "/cygdrive/c/tools/mingw#{arch}/bin:$(PATH):/cygdrive/c/ProgramData/chocolatey/tools")
+    pkg.environment("CYGWIN", "nodosfilewarning winsymlinks:native")
+
+    #pkg.build_requires "pl-iconv-#{platform.architecture}"
+
+    pkg.build do
+      ":"
+    end
+
+    pkg.install do
+      "cp -r * #{settings[:prefix]}"
```

Comment:
If you want to spend some time looking at that with me, I'm game. But I didn't want to spend more than a couple hours on it, because there's a lot planned for Glisan.